test/string-class-test.js

```diff
@@ -0,0 +1,175 @@
+var expect = require('chai').expect;
+require('../src/string-class');
+
+describe('String Class', () => {
+
+  describe('hasVowels', () => {
+    it('should return true for strings that has vowels in them', () => {
+      expect('word'.hasVowels()).to.be.true;
+      expect('andela'.hasVowels()).to.be.true;
+    });
+    it('should return true for strings that has vowels no matter the case of the word', () => {
+      expect('AndEla'.hasVowels()).to.be.true;
+    });
+    it('should return false for strings with consonants only', () => {
+      expect('cyst'.hasVowels()).to.be.false;
+      expect('Hymn'.hasVowels()).to.be.false;
+    });
+  });
+
+  describe('toUpper', () => {
+    it('should convert strings in lowercase to uppercase', () => {
+      expect('cyst'.toUpper()).to.equal('CYST');
+      expect('andela'.toUpper()).to.equal('ANDELA');
+    });
+    it('should convert strings with title case to uppercase', () => {
+      expect('Cyst'.toUpper()).to.equal('CYST');
+      expect('Andela'.toUpper()).to.equal('ANDELA');
+    });
+    it('should convert strings that are mixed with both lowercase and uppercase to uppercase', () => {
+      expect('cYsT'.toUpper()).to.equal('CYST');
+      expect('AnDelA'.toUpper()).to.equal('ANDELA');
+    });
+    it('uppercase strings should remain in uppercase', () => {
+      expect('ANDELA'.toUpper()).to.equal('ANDELA');
+    });
+  });
+
+  describe('toLower', () => {
+    it('should convert strings in uppercase to lowercase', () => {
+      expect('ANDELA'.toLower()).to.equal('andela');
+      expect('TOMILAYO'.toLower()).to.equal('tomilayo');
+    });
+    it('should convert strings with title case to lowercase', () => {
+      expect('Andela'.toLower()).to.equal('andela');
+      expect('Tomilayo'.toLower()).to.equal('tomilayo');
+    });
+    it('should convert strings that are mixed with both lowercase and uppercase to lowercase', () => {
+      expect('cYsT'.toLower()).to.equal('cyst');
+      expect('AnDelA'.toLower()).to.equal('andela');
+      expect('ToMilaYO'.toLower()).to.equal('tomilayo');
+    });
+    it('lowercase strings should remain in lowercase', () => {
+      expect('andela'.toLower()).to.equal('andela');
+      expect('tomilayo'.toLower()).to.equal('tomilayo');
+    });
+  });
+
+  describe('ucFirst', () => {
+    it('should return the string with the first letter in upper case', () => {
+      expect('AnDelA'.ucFirst()).to.equal('AnDelA');
+      expect('tomilayo'.ucFirst()).to.equal('Tomilayo');
+    });
+    it('should return an unchanged string if first letter is uppercase', () => {
+      expect('Andela'.ucFirst()).to.equal('Andela');
+      expect('Tomilayo'.ucFirst()).to.equal('Tomilayo');
+    });
+  });
+
+  describe('isQuestion', () => {
+    it('should return "true" if the entire string is a question', () => {
+      expect('How are you?'.isQuestion()).to.be.true;
+    });
```

Comment:
this fails when 'How are you?' is written as 'How are you??'. You may want to handle this edge case